packages/p2p-v2/src/hooks/useSendbird.ts

```diff
@@ -221,12 +221,7 @@ const useSendbird = (orderId: string) => {
 
     const initialiseChat = useCallback(async () => {
         try {
-            if (
-                isSuccessSendbirdServiceToken &&
-                isSuccessAdvertiserInfo &&
-                sendbirdServiceToken?.app_id &&
-                advertiserInfo?.chat_user_id
-            ) {
+            if (isSuccessSendbirdServiceToken && sendbirdServiceToken?.app_id && advertiserInfo?.chat_user_id) {
```

Comment:
no need to check for isSubscribed, since checking for `advertiserInfo.chat_user_id` is already enough